### PR TITLE
Pla 8346 doc typos

### PR DESCRIPTION
**Pull request by talltea**
# Citrine Python PR

## Description 
Small typo fixes in the docs

### PR Type:
- [ ] Breaking change (fix or feature that would cause existing functionality to change)
- [x] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)

### Adherence to team decisions
- [ ] I have added tests for 100% coverage
- [ ] I have written Numpy-style docstrings for every method and class.
- [ ] I have communicated the downstream consequences of the PR to others.
- [ ] I have bumped the version in setup.py

### CODE REVIEW

File: docs/source/getting_started/code_examples.rst [hunk 1]
```diff
@@ -121,7 +122,7 @@ a parameter template ``wavelength_template``, and a property template ``refracti
     reaction_run = solvents_dataset.process_runs.register(
         ProcessRun("A chemical reaction", spec=reaction_spec))
     toluene_ingredient = solvents_dataset.ingredient_runs.register(
-        IngredientRun("Toluene solvent", spec=toluene_ingredient_spec,
+        IngredientRun(spec=toluene_ingredient_spec,
         material=toluene, process=reaction_run, absolute_quantity=NominalReal(40, 'mL'), notes="I poured too much!"))
 
 Getting a material history
```
Comment (talltea) [inline docs/source/getting_started/code_examples.rst:128]: @kroenlein do you know what a `buy_toluene` is supposed to refer to on line https://github.com/CitrineInformatics/citrine-python/pull/687/files#diff-daad57bd0515ee9846f70e7f9321bea433d4f17671e62b334ffbf76292cac928R148 below?

Comment (kroenlein) [reply docs/source/getting_started/code_examples.rst:128]: It sounds like `buy_toluene` is supposed to the the process spec that generates the raw material `Toluene`.  Is that the question?

File: docs/source/getting_started/projects.rst [hunk 2]
```diff
@@ -124,4 +124,4 @@ A user's role in a Project can be updated. The method
     project = citrine.projects.register(name="Your Project")
 
     # Make the user a member with write access
-    project.update_user_role(user_id=user_id, role=MEMBER, actions=[WRITE])
+    project.update_user_role(user_uid=user_uid, role=MEMBER, actions=[WRITE])
```
Comment (kroenlein) [inline docs/source/getting_started/projects.rst:127]:
```suggestion
    project.update_user_role(user_uid=user_id, role=MEMBER, actions=[WRITE])
```
`user_uid` is not defined above.  `user_id` is though.  Did you want to just `s/_id/_uid/g`?

**Review by kroenlein — CHANGES_REQUESTED** [review]
Good catches.

**Review by kroenlein — APPROVED** [review]

**Pull request merged**